src/csharp/Extensions/Microsoft.Spark.Extensions.Delta.E2ETest/DeltaTableTests.cs

```diff
@@ -276,6 +286,9 @@ public void TestSignatures()
             Assert.IsType<DataFrame>(table.Vacuum());
             Assert.IsType<DataFrame>(table.Vacuum(168));
 
+            // Generate should return void.
+            table.Generate("symlink_format_manifest");
```

Comment:
Is it possible to have a functional test for `Generate()`?

Reply:
Make sense. Will do. Thanks!

Reply:
Generally, we don't need functional tests. @AFFogarty why do you suggest having a functional test?

Reply:
We do have basic functional tests for some of the APIs:  https://github.com/dotnet/spark/blob/master/src/csharp/Extensions/Microsoft.Spark.Extensions.Delta.E2ETest/DeltaTableTests.cs#L32

However, I think that it's OK if we skip the functional test for now (maybe add a TODO)?  I'd rather prioritize getting this PR merged as soon as possible.

Reply:
That looks like a scenario test. If we want to do functional tests, we should do it for all, not just few. Let's follow how E2E test is done for Microsoft.Spark for the time being.

Reply:
So I will keep as it is for now. Let's revisit if needed. Is that fine @AFFogarty @imback82 ?